### PR TITLE
V3.8.6

### PR DESCRIPTION
**Pull request by ocefpaf**
@mingwandroid I used your workflow to handle the patches here. Do you mind taking a look?

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -268,6 +270,7 @@ about:
   dev_url: https://docs.python.org/devguide/
 
 extra:
+  feedstock-name: python
```
Comment (ocefpaf) [inline recipe/meta.yaml:273]: @xhochy I incorporated https://github.com/conda-forge/python-feedstock/pull/393 here, is that OK?

**Comment by mingwandroid** [comment]
I also have git-format-patch-le.py which is a little better than make-mixed in that it can emit patch files with the correct line endings when those vary between files. Still, @jjhelmus has good luck with `git config --global core.autocrlf false` for regenerating patches like this. I never had any luck and false is the default for that and I'm not sure it does anything on non-Windows.

File: recipe/meta.yaml (hunk 1 above)
Comment (xhochy) [reply recipe/meta.yaml:273]: Yes, sure!

**Comment by ocefpaf** [comment]
> I also have git-format-patch-le.py which is a little better than make-mixed in that it can emit patch files with the correct line endings when those vary between files.

I have both scripts here. Thanks! I'm just experimenting for now. But good to know that `git-format-patch-le.py` is better.

> Still, @jjhelmus has good luck with `git config --global core.autocrlf false` for regenerating patches like this. I never had any luck and false is the default for that and I'm not sure it does anything on non-Windows.

Yep. I forgot to do that. But I'm adding it to the docs I'm writing.

**Comment by jjhelmus** [comment]
LGTM

**Review by jjhelmus — APPROVED** [review]

**Comment by jjhelmus** [comment]
Might be good change this PR to merge into the `3.8` branch so that #395 does not need to be re-based.

**Comment by ocefpaf** [comment]
> Might be good change this PR to merge into the `3.8` branch so that #395 does not need to be re-based.

Indeed. Let me fix that.

**Comment by ocefpaf** [comment]
Thanks @jjhelmus for the review! And thanks you both for teaching me something new! I now (kind of) know how to manage these patches.

**Pull request merged**